### DISCUSSION TITLE
Probably a timeline of the next few versions of the stable firmwares

### DISCUSSION THREAD

**Discussion by jasperchua99** [egzumer/uv-k5-firmware-custom#386] [General]
hi possible to publish what is the roadmap for the project and probably indicate when the next version of the firmware be released?

**Comment by ErikS-web** [comment]
This is all Hobby and Fun and spare time. 
So NO planning and NO pressure at all.

But we are open for good idea's to get the most functionality out of the QS

**Comment by egzumer** [comment]
This might not look like it is the case, I'm only as good of a programmer, but the project was like a fulltime job for some time, just without a salary 🤣, and I'm getting tired of it, so there will be tuning back rather than full steam ahead. 
There is nothing new in the FW. I was working on the chirp driver and calibration editor, I will publish that soon. Other than that there is no roadmap and there won't be.
You can take a look at what other forks of my firmware introduce if you are hungry of further improvements, there are some interesting mods there:
https://gitpop2.vercel.app/egzumer/uv-k5-firmware-custom

Looks like [losehu](https://github.com/losehu/uv-k5-firmware-egcn) is ahead of the pack. 
[kamilsss655](https://github.com/kamilsss655/uv-k5-firmware-custom) is a HAM (I'm not) and he introduced some improvements that go towards that community
[joaquimorg](https://github.com/joaquimorg/uv-k5-firmware-custom) introduced his messenger mod, worth checking out.

**Reply by WHIZZKIIDD** [reply]
Can i ask ? I am totally new here, have no software experience but can someone tell me, is it possible to change the default register values that determine gains, filters etc ?  I would love to play with different settings ... but how ? Do I have to learn Python ?
